be/.env.example

```diff
@@ -1,8 +1,19 @@
-Here is example .env file
+#Here is example .env file
 
+#example database ip address
+DATABASE_IP=0.0.0.0
```

Comment:
Maybe change to `127.0.0.1.` so it would immediately work with the docker db setup